Source/Core/Core/HW/DVDInterface.cpp

```diff
@@ -1291,7 +1343,7 @@ u64 SimulateDiscReadTime(u64 offset, u32 length)
 		}
 	}
 
-	g_last_read_offset = (offset + length - 2048) & ~2047;
+	g_last_read_offset = end_offset;
```

Comment:
Actually, screw it. Keeping the "- 2048" where it was is probably the best. I'll add something separate for buffering decrypted reads later.